### PR TITLE
add emoji support to live app

### PR DESCRIPTION
**Pull request by olipyskoty**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### JIRA link(s):

- [VIDEO-10996](https://issues.corp.twilio.com/browse/VIDEO-10996)

### Description

This PR adds a few emoji buttons to the chat window. Open to changing the emojis if we want! I also removed the `<Snackbar />` component since there will never be any file sending errors.

Screen Shot 2022-08-08 at 12 53 56 PM

### Before review
* [ ] Added unit tests if necessary
* [ ] Updated affected documentation
* [x] Verified locally with all effected platforms
* [x] Manually sanity tested running locally
* [x] Included screenshot as PR comment (if needed)
* [x] Ready for review

### Before merge
* [ ] Got one or more +1s
* [ ] Re-tested if necessary

### CODE REVIEW

**Comment by olipyskoty** [comment]
@timmydoza added functionality for increased font size when message is only one emoji

Screen Shot 2022-08-09 at 1 17 33 PM

**Review by timmydoza — CHANGES_REQUESTED** [review]

**Comment by timmydoza** [comment]
@olipyskoty Looks great! My only thought here is that the `isOnlyEmojis` boolean is something that it would be nice to have tests for. Do you think it would be easy to use [this](https://github.com/twilio/twilio-video-app-react/blob/master/src/components/ChatWindow/MessageList/MessageList.test.tsx) and add an emoji message to it (and get rid of the file messages)?

**Comment by olipyskoty** [comment]
@timmydoza I added that snapshot test to this repo which does confirm that the `isOnlyEmojis` class name gets added. Let me know if you wanted a specific test for that boolean!

File: apps/web/src/components/ChatWindow/ChatInput/ChatInput.tsx [hunk 1]
```diff
@@ -32,8 +32,10 @@ const useStyles = makeStyles(theme => ({
     },
   },
   buttonContainer: {
-    margin: '1em 0 0 1em',
+    margin: '1em 0 0 0.5em',
```
Comment (timmydoza) [inline apps/web/src/components/ChatWindow/ChatInput/ChatInput.tsx:35]: My only thought here is - should there be 0 left-margin? Just thinking it looks the tiniest bit lop-sided with left-margin and no right-margin. What do you think?
image
image

Comment (olipyskoty) [reply apps/web/src/components/ChatWindow/ChatInput/ChatInput.tsx:35]: Yep that looks good!

**Review by timmydoza — APPROVED** [review]

**Pull request merged**